src/content/docs/en/reference/errors/session-config-missing-error.mdx

```diff
@@ -13,12 +13,16 @@ import DontEditWarning from '~/components/DontEditWarning.astro'
 <DontEditWarning />
 
 
-> The `experimental.session` flag was set to `true`, but no storage was configured. Either configure the storage manually or use an adapter that provides session storage
+:::caution[Deprecated]
+This error was removed in Astro 5.7, when the Session feature stopped being experimental.
+:::
```

Comment:
Generally, we refer to feature names in plural, so Action**s**, View Transition**s**, etc. So when referring to the feature here, "Sessions" should be used to keep that same pattern.
```suggestion
:::caution[Deprecated]
This error was removed in Astro 5.7, when the Sessions feature stopped being experimental.
:::
```

Reply:
Will this need to be edited at source?

Reply:
Are deprecated errors removed from the errors JSDoc stuff? I am not sure. If they're removed, the CI would probably keep the pages as they are, if not them yeah, probably something we would have to deal at the source.